### PR TITLE
add storage file

### PR DESCRIPTION
**Pull request by rmjspecial**
**Project-Description**

- Add footer method to clear all computed tasks.
- Add an update storage message to clear all computed storage.

### CODE REVIEW

File: src/clear.js [hunk 1]
```diff
@@ -12,4 +12,8 @@ const clearAll = () => {
       deleteTask(e);
     }
   });
+  let count = 0;
+  const data = Array.from(localData).filter((e) => e.completed === false);
+  data.map((e) => e.index = count += 1);
+  setStorage(data);
 };
```
Comment (medaminedev66) [inline src/clear.js:19]: - [x] The user should get the correct list values after the page reloads. Please make sure when you reload the page the checkboxes next to the tasks are checked if the tasks are completed.

File: src/clear.js [hunk 2]
```diff
@@ -12,4 +12,8 @@ const clearAll = () => {
       deleteTask(e);
     }
   });
+  let count = 0;
+  const data = Array.from(localData).filter((e) => e.completed === false);
+  data.map((e) => e.index = count += 1);
+  setStorage(data);
```
Comment (medaminedev66) [inline src/clear.js:18]: - [x] When I reload the page and try to change the checkbox of a task, the completed value of all the other tasks changes to false in the local storage. Please correct that.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @rmjspecial  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- There are no linter errors
- All the completed tasks are removed after clicking on the clear all completed tasks button.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rmjspecial** [comment]
fix all issues according to the requirements.

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 

Hi @rmjspecial , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**